### ISSUE TITLE
Does not work with attributes with underscore in their name

### ISSUE THREAD

**Issue by sahin52** [Box-Of-Hats/vscode-ts-to-cs#13]
#### Example:  
```
export interface TempInterface {
    tempvar: string
  }
```
after Ctrl+Shift+P and selecting `TS=>CS....` in this file, result is a new file with:   
```
public class TempInterface {
    
    [JsonProperty("tempvar")]
    public string Tempvar;
    
}
```
text in it.  
#### Example:
```
export interface TempInterface {
    temp_var: string
  }
```
after Ctrl+Shift+P and selecting `TS=>CS....` in this file a warning is shown with `Could not find any exported interfaces to generate from.` text on it.

The only difference is an underscore (`_`)

**Comment by sahin52** [comment]
I guess I resolved it, in https://github.dev/Box-Of-Hats/ts-csharp in index.ts file the regexes does not have underscore

**Comment by sahin52** [comment]
I created a pull request for it. https://github.com/Box-Of-Hats/ts-csharp/pull/12

**Comment by Box-Of-Hats** [comment]
Thanks again for the PR 🚀

I've bumped the version of ts-csharp used by the extension and released [v0.2.4](https://github.com/Box-Of-Hats/vscode-ts-to-cs/releases/tag/v0.2.4) so this should now be working as expected
